src/components/interaction-designer/toolbar/ErrorNotifications.vue

```diff
@@ -103,16 +107,18 @@ export class ErrorNotifications extends mixins(Routes, Lang) {
   get blockValidationStatuses(): { [key: string]: IValidationStatus } {
     const blocksMap = this.activeFlow?.blocks.reduce((map: { [key: string]: boolean }, block) => {
       map[`block/${block.uuid}`] = true
+      map[`backend/block/${block.uuid}`] = true
       return map
     }, {})
     if (!blocksMap || size(blocksMap) === 0) {
       return {}
     }
+    // frontend or backend validation keys have `block/` string
     return pickBy(this.validationStatuses, (value: IValidationStatus, key) => key.includes('block/') && blocksMap[key])
   }
 
   hasBlockValidationErrors(uuid: string): boolean {
-    return has(this.blockValidationStatuses, `block/${uuid}`)
+    return has(this.blockValidationStatuses, `block/${uuid}`) || has(this.blockValidationStatuses, `backend/block/${uuid}`)
```

Comment:
making sure we consider the `backend` prefixed keys, even if there is no frontend validation errors.